hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```
@@ -470,6 +475,24 @@ objects:
       mappingMethod: ${IDENTITY_MAPPING_METHOD}
       name: ${IDENTITY_NAME}
       type: LDAP
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncIdentityProvider
+  metadata:
+    name: osd-sre-google-identityprovider
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+        api.openshift.com/environment: production
+    identityProviders:
+    - google: null
+      clientID: 211953714079-afmg28lojv8m2vv97b56s64ivm5ercn4.apps.googleusercontent.com
+      clientSecret:
+        name: osd-google-secret
+      hostedDomain: redhat.com
```

```suggestion
    - google:
        clientID: ${IDENTITY_SRE_GOOGLE_CLIENTID}
        clientSecret:
          name: osd-google-secret
        hostedDomain: redhat.com
```

Indentation, value for `google` and move clientID to parameter.  The param will need to come in an update, I can't add a suggestion sorry.